Xamarin.Forms.Platform.UAP/VisualElementPackager.cs

```diff
@@ -127,7 +119,7 @@ void OnChildAdded(object sender, ElementEventArgs e)
 
 			_panel.Children.Add(childRenderer.ContainerElement);
 
-			if (_isZChanged)
+			if (ElementController.LogicalChildren[ElementController.LogicalChildren.Count - 1] != view)
```

Comment:
What is the purpose of this test? It causes the Z-Index of the last element to be set to zero (rendering it under every other element, while it should be rendered on top of every other element). See issue #2943